two minor bugfixes

murrahjm opened the pull request:
Line 210 looks like a typo, was appending ') to the end of the designer filename.

Line 100 was passing the color as unquoted text to from argb.  So:
  [system.drawing.color]::FromArgb(red) in my case.  I guess powershell studio isn't returning the actual rgb code.  Added code to compute that from the color text and insert that instead.

murrahjm committed:
two minor bugfixes

adamdriscoll commented on utilities/ConvertTo-PoshTools.ps1:
```diff
@@ -97,7 +97,8 @@ function Out-ObjectProperty {
         }
         elseif ($propertyType.FullName -eq "System.Drawing.Color") {
-            $value =  "[System.Drawing.Color]::FromArgb($($property.InnerText))"
+            $color =  "[System.Drawing.Color]::$($property.InnerText)"
```
This will return an array of ints if the color is a custom color so we would probably need a way to figure that out.

adamdriscoll reviewed (changes requested):
We probably need to handle the situation where a custom color is passed into the color property. It will look like: 12, 12, 12

murrahjm committed:
Update ConvertTo-PoshTools.ps1

in addition to the color object and the custom color array, there's the third option of a systemcolors object.  This edit will try to cast the value as systemcolors, then colors, then finally just create a color object from a custom array.  This preserves the class type in visual studio which looks like it might be important for the systemcolors stuff

adamdriscoll reviewed (approved):
Looks great!
